ukwa/npld-player
# Launch app using custom protocol

(https://github.com/ukwa/npld-player/issues/14) Launches app by clicking on links using custom `npld-viewer` protocol.

Tested on Mac OSX 12.1.

### Manual testing
1. Build package with `yarn make`. Open `npld-player` app in `out/npld-player-{build}`
2. Open `test/index.html` in your browser. Click "Open example.com in app". Verify you see a dialog asking to open link in `npld-player`
3. Confirm dialog. Verify npld-player app window focuses and loads URL

### Screenshots
On Chrome v100:
<img width="751" alt="Screen Shot 2022-05-23 at 3 36 41 PM" src="https://user-images.githubusercontent.com/4672952/169915831-8248745e-c181-482d-b92c-81056e5752c0.png">
